Seminar/sem_5/26.py

```diff
@@ -0,0 +1,18 @@
+# Задача 26:  Напишите программу, которая на вход принимает два числа 
+# A и B, и возводит число А в целую степень B с помощью рекурсии.
+
+# *Пример:*
+
+# A = 3; B = 5 -> 243 (3⁵)
+#     A = 2; B = 3 -> 8 
+
+def recursion_degree(B):
+    if B == 1:
+        return A
+    else:
+        return A * recursion_degree(B - 1)

```

Comment:
Блок кода `else` не обязателен.
```suggestion
    return A * recursion_degree(B - 1)

```